### PR TITLE
[StorageMover] Initalize Track2 MGMT SDK

### PR DESCRIPTION
**Pull request by HarveyLink**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/storagemover/Azure.ResourceManager.StorageMover/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Release History
```
Comment (Yao725) [inline sdk/storagemover/Azure.ResourceManager.StorageMover/CHANGELOG.md:1]: It seems that the CHANGELOG and README files are out of date and don't follow our requirement, could you please take a look at this [PR](https://github.com/Azure/azure-sdk-for-net/pull/32935) and update the files correspondingly?

File: sdk/storagemover/Azure.ResourceManager.StorageMover/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("StorageMover")]
```
Comment (Yao725) [inline sdk/storagemover/Azure.ResourceManager.StorageMover/src/Properties/AssemblyInfo.cs:6]: I assume here should be `Microsoft.StorageMover`

**Review by Yao725 — APPROVED** [review]

File: sdk/storagemover/Azure.ResourceManager.StorageMover/src/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (HarveyLink) [reply sdk/storagemover/Azure.ResourceManager.StorageMover/src/Properties/AssemblyInfo.cs]: Done

File: sdk/storagemover/Azure.ResourceManager.StorageMover/CHANGELOG.md (hunk 1 above)
Comment (HarveyLink) [reply sdk/storagemover/Azure.ResourceManager.StorageMover/CHANGELOG.md:1]: Done

File: sdk/storagemover/Azure.ResourceManager.StorageMover/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## 1.0.0-beta.1 (Unreleased)
```
Comment (ArthurMa1978) [inline sdk/storagemover/Azure.ResourceManager.StorageMover/CHANGELOG.md:3]: Specify the release day that we can release the beta package after merged

**Review by ArthurMa1978 — APPROVED** [review]

File: sdk/storagemover/Azure.ResourceManager.StorageMover/CHANGELOG.md (hunk 3 above)
Comment (HarveyLink) [reply sdk/storagemover/Azure.ResourceManager.StorageMover/CHANGELOG.md]: Done

**Comment by blueww** [comment]
@HarveyLink 
Would you please indicate this PR's goal?

The SDK code in this PR looks is on the preview API version 2022-07-01-preview.
However, there's a stable API version swagger PR for StorageMover is in review https://github.com/Azure/azure-rest-api-specs-pr/pull/9785

As we discussed with the feature PM before, StorageMover GA will need .net SDK support, and should be based on the stable API version.

**Pull request merged**